src/Microsoft.AspNetCore.Razor.Language/DefaultRazorIRLoweringPhase.cs

```diff
@@ -17,6 +17,9 @@ protected override void ExecuteCore(RazorCodeDocument codeDocument)
             var syntaxTree = codeDocument.GetSyntaxTree();
             ThrowForMissingDependency(syntaxTree);
 
+            var tagHelperContext = codeDocument.GetTagHelperContext();
```

Comment:
Can this method be renamed to `GetTagHelperDocumentContext()`? We already have a `TagHelperContext` in `Razor.Runtime` that is more widely known. This could cause some confusion.